src/key_value/key_value_test.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+#include "key_value.h"
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+namespace {
+
+using csci499::KeyValue;
+
+// test initialization of csci499
+TEST(KeyValue, OneVectOneValue) {
+  KeyValue test;
+  test.Put("a", "0");
+  EXPECT_EQ(test.Get("a")[0], "0");
+}
+
+TEST(KeyValue, OneVectTwoValues) {
+  KeyValue test;
+  test.Put("a", "0");
+  test.Put("a", "1");
+  EXPECT_EQ(test.Get("a")[0], "0");
+  EXPECT_EQ(test.Get("a")[1], "1");
+}
+
+TEST(KeyValue, TwoVectTwoTwoValues) {
+  KeyValue test;
+  test.Put("a", "0");
+  test.Put("a", "1");
+  test.Put("b", "2");
+  test.Put("b", "3");
+  EXPECT_EQ(test.Get("a")[0], "0");
```

Comment:
You may want an ASSERT before these EXPECTs because it could crash if the number of values doesn't match what you want.

Reply:
So should all of the EXPECTs be ASSERT for only this test or all of the tests?